client/reader/site-stream/featured.jsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import page from 'page';
+import PostStore from 'lib/feed-post-store';
+import FeedPostStoreActions from 'lib/feed-post-store/actions';
+import stats from 'reader/stats';
+
+export default React.createClass( {
+	displayName: 'FeedFeatured',
+
+	getInitialState() {
+		return this.getStateFromStores();
+	},
+
+	getStateFromStores( store ) {
+		store = store || this.props.store;
+		return {
+			posts: store.get()
+		};
+	},
+
+	updateState( store ) {
+		this.setState( this.getStateFromStores( store ) );
+	},
+
+	componentDidMount() {
+		this.props.store.on( 'change', this.updateState );
+		PostStore.on( 'change', this.updateState );
+	},
+
+	componentWillUnmount() {
+		this.props.store.off( 'change', this.updateState );
+		PostStore.off( 'change', this.updateState );
+	},
+
+	handleClick( post ) {
+		stats.recordAction( 'clicked_featured_post' );
+		stats.recordGaEvent( 'Clicked Featured Post' );
```

Comment:
need to record an event to tracks here too. probably stats.recordTrack( 'calypso_reader_clicked_featured_post', { blog_id: post.site_ID, post_id: post.ID } ) ...

would be nice to include the source stream in the track, but that would mean you'd have to bump this tracking out to the stream and pass in a callback to notify the stream when the click happened